apps/nextjs/src/app/[locale]/manage/search-engines/_form.tsx

```diff
@@ -25,13 +28,16 @@ export const SearchEngineForm = (props: SearchEngineFormProps) => {
   const { submitButtonTranslation, handleSubmit, initialValues, isPending, disableShort } = props;
   const t = useI18n();
 
+  const { data: integrationData } = clientApi.integration.allThatSupportSearch.useQuery();
```

Comment:
Maybe we can use `useSuspenseQuery` instead? Then it's already loaded when you see the form, otherwise add a loading spinner